data/analytics/events.yml

```diff
@@ -413,17 +413,46 @@
           value: generic download
         - name: url
     - name: mailto links
-      implemented: false
+      implemented: true
       priority: low
+      description: 'When a link points to an email address using mailto:'
+      tracker: specialist_link_tracker.
       data:
       - name: event
         value: event_data
+      - name: event_data
+        value:
+        - name: event_name
+          value: navigation
+        - name: external
+          value: 'string (true)'
+        - name: link_domain
+        - name: link_path_parts
+        - name: method
+        - name: text
+        - name: type
+          value: email
+        - name: url
     - name: attachment links
       implemented: false
```

Comment:
Is this not implemented yet? I thought it was?

Reply:
It hasn't been deployed to production yet, it requires republishing all the guidance documents, and the PAs are changing it from just `html attachment` to `attachment`, so I'm not sure how long it'll be before it's fully implemented.

Reply:
Fair enough, thanks.